packages/quantic/force-app/main/default/lwc/quanticRadioButtonsGroup/quanticRadioButtonsGroup.js

```diff
@@ -0,0 +1,160 @@
+ */
+
+export default class QuanticRadioButtonsGroup extends LightningElement {
+  /**
+   * The value of the label for the Radio Buttons Group.
+   * @api
+   * @type {String}
+   */
+  @api legend = '';
+  /**
+   * The list of options to be displayed.
+   * @api
+   * @type {QuanticRadioButtonsGroupOption[]}
+   */
+  @api options;
+  /**
+   * If the labels should be hidden or visible.
+   * @api
+   * @type {Boolean}
+   */
+  @api hideLabels;
+  /**
+   * The value of the selected radio button inside the Radio Buttons Group.
+   * @api
+   * @type {String}
+   */
+  @api get value() {
+    return this._value;
+  }
+  set value(value) {
+    this._value = value;
+  }
+  /**
+   * The size of the icon.
+   * @api
+   * @type {'xx-small' | 'x-small' | 'small' | 'medium' | 'large'}
+   */
+  @api
+  get iconSize() {
+    return this._iconSize;
+  }
+  set iconSize(value) {
+    if (['xx-small', 'x-small', 'small', 'medium', 'large'].includes(value)) {
+      this._iconSize = value;
+    }
+  }
+
+  @track _value;
+
+  /** @type {'xx-small' | 'x-small' | 'small' | 'medium' | 'large'} */
+  _iconSize = 'xx-small';
+
+  get radioButtonElements() {
+    return this.template.querySelectorAll('input');
+  }
+
+  get transformedOptions() {
+    const {options, value} = this;
+    if (Array.isArray(options)) {
+      return options.map((option, index) => ({
+        label: option.label,
+        value: option.value,
+        iconName: option.iconName,
+        iconVariant: value === option.value ? 'inverse' : '',
+        isChecked: value === option.value,
+        buttonCSSClass: this.makeRadioButtonCSSClass(option),
+        iconCSSClass: this.makeIconCSSClass(option),
+        indexId: `radio-${index}`,
+        tooltip: option.tooltip,
+        mouseEnterHandler: () => {
+          const tooltipToShow = this.getTooltipComponentByValue(option.value);
+          tooltipToShow.showTooltip();
+        },
+        mouseLeaveHandler: () => {
+          const tooltipToHide = this.getTooltipComponentByValue(option.value);
+          tooltipToHide.hideTooltip();
+        },
+      }));
+    }
+    return [];
+  }
+
+  get computedLabelClass() {
+    return `slds-radio_button__label radio-buttons-group__label ${this.hideLabels ? 'slds-assistive-text' : ''}`;
+  }
+
+  makeIconCSSClass(option) {
+    return `radio-button-icon${option.value === this.value ? '--selected' : ''}`;
+  }
+
+  makeRadioButtonCSSClass(option) {
+    let paddingClass = '';
+    if (this.hideLabels && option.iconName) {
+      paddingClass = 'slds-var-p-horizontal_x-small';
+    } else if (option.iconName) {
+      paddingClass = 'slds-var-p-left_medium';
+    }
+    return `slds-button slds-radio_button ${paddingClass} radio-button radio-button--${option.value === this.value ? 'selected' : 'unselected'}`;
+  }
+
+  handleFocus() {
+    this.dispatchEvent(new CustomEvent('focus'));
+  }
+
+  handleBlur() {
+    this.dispatchEvent(new CustomEvent('blur'));
+  }
```

Comment:
This comes from the base implementation of the radio-buttons-group from the LWC components library.
I think this was mostly there so we could handle "dirty" data, in the sense if those radio buttons were in a form that would have validation after you change a value for example.

They are not really useful here with the current usage of the radio-buttons-group. It's more of a "could be useful later"

Reply:
It's not clear for me how this could be concretely useful  in our use case, so I would suggest to remove it unless you do have a clear idea of how we could benefit from this later in the futur.

Cause I think this is typically the things we add  and their purpose is not really clear and in the future we will be always afraid of removing them fairing to break something.

Reply:
yea good point 👍